docs/why-netdata/README.md

```diff
@@ -0,0 +1,30 @@
+# Why Netdata
+
+> Any performance monitoring solution that does not go down to per second
+> collection and visualization of the data, is useless.
+> It will make you happy to have it, but it will not help you more than that. 
+
+Netdata is built around 4 principles:
+
+1. **[Per second data collection for all metrics.](1s-granularity.md)**
+
+    *It is impossible to monitor a 2 second SLA, with 10 second metrics.*
+
+2. **[Collect and visualize all the metrics from all possible sources.](unlimited-metrics.md)**
+
+    *To troubleshoot slowdowns, we need all the available metrics. The console should provide nothing more.*
```

Comment:
`The console should provide nothing more` <- what does this mean?

Reply:
"The console should not provide more metrics"
fixed it